tests/Doctrine/Tests/ORM/Mapping/ClassMetadataTest.php

```diff
@@ -127,15 +127,6 @@ public function testFieldIsNullableByType(): void
         // Explicit Not Nullable
         $cm->mapField(['fieldName' => 'username', 'length' => 50]);
         $this->assertFalse($cm->isNullable('username'));
-
-        // Join table Nullable
-        $cm->mapOneToOne(['fieldName' => 'email', 'joinColumns' => [[]]]);
-        $this->assertFalse($cm->getAssociationMapping('email')['joinColumns'][0]['nullable']);
-        $this->assertEquals(CmsEmail::class, $cm->getAssociationMapping('email')['targetEntity']);
-
-        $cm->mapManyToOne(['fieldName' => 'mainEmail']);
-        $this->assertTrue($cm->getAssociationMapping('mainEmail')['joinColumns'][0]['nullable']);
-        $this->assertEquals(CmsEmail::class, $cm->getAssociationMapping('mainEmail')['targetEntity']);
```

Comment:
I believe `targetEntity` tests should stay. My method naming is lacking here as it suggests only checks for `nullable`, while in fact there are 2 checks for `targetEntity` also. Probably method name change would be good?